08-transport.md

```diff
@@ -371,13 +364,11 @@ for the `poly1305` tag.
   * `ck, temp_k = HKDF(ck, ss)`
      * This phase generates a new temporary encryption key (`temp_k`) which is
        used to generate the authenticating MAC.
-     * The nonce `n` should be reset to zero: `n = 0`.
 
 
-  * `p = decryptWithAD(temp_k, n, h, c)`
+  * `p = decryptWithAD(temp_k, 1, h, c)`
     * If the MAC check in this operation fails, then the initiator MUST
       terminate the connection without any further messages.
-    * The nonce `n` should be reset to zero: `n = 0`.
```

Comment:
Yeah, so this line (the nonce reset after decrypt) shouldn't be there, nice catch!